templates/helm-values-dashboard.tpl.yaml

```diff
@@ -0,0 +1,10 @@
+resources:
+  limits:
+    cpu: 100m
+    memory: 100Mi
+  requests:
+    cpu: 100m
+    memory: 50Mi
+
+ingress:
```

Comment:
this is false by default I believe

Reply:
Yes correct, I should've explained this change. It is false by default, but I added it explicitly in case the default should change, since we never want a direct ingress set normally.

It's also set `false` for our other dashboards.